openfisca_france/reforms/plf2015.py

```diff
@@ -55,12 +55,11 @@ def modify_legislation_json(reference_legislation_json_copy):
     return reference_legislation_json_copy
 
 
-class decote(DatedVariable):
+class decote(Variable):
     label = u"Décote IR 2015 appliquée sur IR 2014 (revenus 2013)"
     definition_period = YEAR
 
-    @dated_function(start = date(2013, 1, 1), stop = date(2013, 12, 31))
```

Comment:
We are in a reform, and in that case, the end doesn't mean that the formula stops in `'2013-12-31'`.

It means that we are replacing the formula of `decote` from `'2013-01-01'` to `'2013-12-31'` (it took me some time to understand, as I had errors when putting an end date).

The behaviour here is slightly different than it was before, as we are now changing the formula "from 2013" instead of "for 2013 only".

It's probably alright in this context (I doubt we'll ever study the impact of the PLF 2015 in 2017), and we don't have a better helper :(.